public/css/svg.css

```diff
@@ -22,6 +22,22 @@
   stroke-dasharray: 5,5;
 }
 
+.hit-links {
+  float: right;
+}
+
+.kablammo {
```

Comment:
Put `grapher` class on the parent `div` of `ref=svgContainer` and apply this style to `.grapher` to get the effect for all visualisation instances at once.